config/pfs/constructNormFlat.py

```diff
@@ -0,0 +1,11 @@
+config.darkTime = None
+
+config.isr.doBias = True
+config.isr.doDark = False
+config.isr.doFlat = False
+#config.isr.doLinearize = False
+
+config.repair.cosmicray.nCrPixelMax = 5000000
```

Comment:
That's a lot of CRs!  Why so many?

What PSF are you guessing when detecting CRs?